### PR TITLE
feat: added avatar and badge component

### PR DESCRIPTION
**Pull request by amirnathani9**
# Change logs
## Gif
avatar
badge

## Preview Link
https://deploy-preview-1--nomark-ui.netlify.app/

### CODE REVIEW

**Review by zabihshaik — APPROVED** [review]
Everything looks Good. And it's responsive as well. 
Code Snippet is also responsive.

**Comment by arjunbharti** [comment]
Neat and clean. The naming convention is also good and easy to understand. Go ahead and merge!

File: components/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-md m-4">
+                    </figure>
+                    <figure>
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-lg m-4">
+                    </figure>
+                    <figure>
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-xl m-4">
+                    </figure>
+                    <figure>
+                        <img src="/assests/avatar.jpg" alt="" class="avatar-xxl m-4">
```
Comment (devdeadviz) [inline components/avatar/avatar.html]: Provide alternate text with respect to your image.

File: components/badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+</head>
+
+<body>
+    <section class="section-center">
+        <!-- navbar -->
+        <nav class="navbar flex items-center justify-between pt-1 section-center z-5">
+            <a class="humburger-icon icon-open text-lg"><i class="fas fa-bars ml-5"></i></a>
+            <a class="humburger-icon icon-close text-lg displayNone"><i class="fas fa-bars ml-5"></i></a>
+            <a href="/index.html" class="mr-5"><img src="/assests/logo-primary.png" alt="NOMARK"
+                    class="nomark-logo"></a>
```
Comment (devdeadviz) [inline components/badge/badge.html]: Fix Indenattion.

File: components/badge/badge.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        <div class="sidebar sidebarRemove flex flex-col scroll-style z-4 pr-7 p-2">
+            <h3 class="sidebar-heading text-lg">Getting Started</h3>
+            <ul>
+                <li><a href="/documentation/doc.html" class="sidebar-items">Installation</a></li>
+            </ul>
+            <h3 class="sidebar-heading text-lg">Components</h3>
+            <ul>
+                <li><a href="/components/avatar/avatar.html" class="sidebar-items">Avatar</a></li>
+                <li><a href="" class="sidebar-items">Alert</a></li>
+                <li><a href="badge.html" class="sidebar-items active">Badge</a></li>
```
Comment (devdeadviz) [inline components/badge/badge.html]: Use proper directory path.

**Review by devdeadviz — COMMENTED** [review]
Add some required changes, you can have a look at it.

File: components/avatar/avatar.html (hunk 1 above)
Comment (amirnathani9) [reply components/avatar/avatar.html]: Done with the alt text

File: components/badge/badge.html (hunk 3 above)
Comment (amirnathani9) [reply components/badge/badge.html]: Given the proper directory path

File: components/badge/badge.html (hunk 2 above)
Comment (amirnathani9) [reply components/badge/badge.html]: fixed

**Pull request merged**